go-controller/pkg/config/config.go

```diff
@@ -95,6 +96,8 @@ type DefaultConfig struct {
 	// ClusterSubnets holds parsed cluster subnet entries and may be used
 	// outside the config module.
 	ClusterSubnets []CIDRNetworkEntry
+	// NbctlDaemonMode enables ovn-nbctl to run in daemon mode
```

Comment:
@girishmg We can remove this block adding the option to the config file since we don't really need it to be passed through the daemonset YAML; we can always hardcode to `true` in the master part of ovnkube.sh

Reply:
done